src/details/ArborX_DetailsNode.hpp

```diff
@@ -23,13 +23,6 @@ namespace ArborX::Details
 
 constexpr int ROPE_SENTINEL = -1;
 
-template <class BoundingVolume>
-struct PairIndexVolume
-{
-  unsigned index;
-  BoundingVolume bounding_volume;
-};
```

Comment:
> Did you do the homework and audited the header includes?

Is there an automatic way to do it? Otherwise, it seems inefficient use of time for little benefit.